lib/add_magic_comment.rb

```diff
@@ -27,18 +28,25 @@ def self.process(options)
 
     count = 0
     EXTENSIONS.each do |ext, comment_style|
-      rbfiles = File.join(directory, "**", "*.#{ext}")
+      case ext
+        when 'erb'
+          comment = "#{MAGIC_COMMENT} %>\n"
+        else
+          comment = "#{MAGIC_COMMENT}\n"
+      end
```

Comment:
This `comment` format is very related to the `EXTENSIONS` hash above. How about making that hash point to an array (or another hash) that has the comment form at it? Then we'll see the symmetry of `<% ... %>`.